src/user-status/userStatusReducers.js

```diff
@@ -0,0 +1,41 @@
+/* @flow strict-local */
+import type { UserStatusState, Action } from '../types';
+import {
+  LOGOUT,
+  LOGIN_SUCCESS,
+  ACCOUNT_SWITCH,
+  REALM_INIT,
+  EVENT_USER_STATUS_UPDATE,
+} from '../actionConstants';
+import { NULL_OBJECT } from '../nullObjects';
+
+const initialState: UserStatusState = NULL_OBJECT;
+
+export default (state: UserStatusState = initialState, action: Action): UserStatusState => {
+  switch (action.type) {
+    case LOGOUT:
+    case LOGIN_SUCCESS:
+    case ACCOUNT_SWITCH:
+      return initialState;
+
+    case REALM_INIT:
+      return action.data.user_status || initialState;
+
+    case EVENT_USER_STATUS_UPDATE: {
+      const newUserStatus = { ...state[action.user_id] } || {};
```

Comment:
Can this `|| {}` ever do anything?

Note that an empty object is already truthy:
```
> ({} || false)
{}
```

Reply:
This code remained from a version that was like `state[action.user_id] || {}`. Fixed.